golem/core/variables.py

```diff
@@ -18,6 +18,7 @@
 PRIVATE_KEY = "keystore.json"
 DEFAULT_PROC_FILE = "node_processes.ctl"
 MAX_PROC_FILE_SIZE = 1024 * 1024
+ENV_TASK_API_DEV = 'ENV_TASK_API_DEV'
```

Comment:
```suggestion
ENV_TASK_API_DEV = 'GOLEM_TASK_API_DEV'
```

I think it makes more sense to not use `ENV` in env, but make it more specific to golem.

Reply:
good one! updated :)